<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>average_window</class>
 <widget class="QWidget" name="average_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define Average Window - ${DEVICE}</string>
  </property>
  <widget class="QLabel" name="start_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>45</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="stop_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>45</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="PyDMWaveformPlot" name="average_window_wf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PV Window Size</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>50</width>
     <height>12</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Write to PV: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_line_edit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter start value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_line_edit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter end value</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: red;}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="PyDMPushButton" name="write_pb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #bc5f6a }</string>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
  <customwidget>
   <class>PyDMWaveformPlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.waveformplot</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
